style.css

```diff
@@ -0,0 +1,100 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+#container {
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  grid-template-rows: repeat(4, 1fr );
+  grid-template-areas:
+        'header header header header header'
+        'mapid  mapid  mapid  notes  links'
+        'mapid  mapid  mapid  notes  links'
+        'mapid  mapid  mapid  notes  links';
```

Comment:
Is there a reason you have 5 columns instead of 3? Also if you're setting the `grid-template-areas` you don't actually need to explicitly state the grid rows and columns.

Reply:
This was done during debugging because the grid wasn't rendering properly so I laid it out this way, then I realized I was using the wrong selector.